text/0265-event-vision.md

```diff
@@ -0,0 +1,75 @@
+# Event Basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to document and agree on a few basics.
+
+### What are OpenTelemetry Events?
+
+OpenTelemetry Events are a type of OpenTelemetry Log that requires an event name and follows a specific structure implied by that event name.
+
+They are a core concept in OpenTelemetry Semantic Conventions.
+
+### OTLP
+
+Since OpenTelemetry Events are a type of OpenTelemetry Log, they share the same OTLP log data structure and pipeline.
+
+### API
+
+OpenTelemetry should provide a (user-facing) Log API that includes the capability to emit OpenTelemetry Events.
+
+### Interoperability with other logging libraries
+
+It should be possible to send OpenTelemetry Logs from the OpenTelemetry Log API to other logging libraries (e.g., Log4j).
+This allows users to integrate OpenTelemetry Logs into an existing (non-OpenTelemetry) log stream.
+
+It should also be possible to bypass the OpenTelemetry Log API entirely and emit OpenTelemetry Logs (including Events)
+directly via existing language-specific logging libraries, if that library has the capability to do so.
+
+OpenTelemetry will recommend that
+[instrumentation libraries](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-library)
+use the OpenTelemetry Log API to emit OpenTelemetry Events rather than using other logging libraries to emit OpenTelemetry Events. This recommendation aims to provide users with a simple and consistent
+onboarding experience that avoids mixing approaches.
+
+OpenTelemetry will also recommend that application developers use the OpenTelemetry Log API to emit OpenTelemetry Events instead of using another
```

Comment:
wondering why is this recommendation required? If the 'other api' has support for emitting structured logs with event name, should Otel have reason to recommend one over the other? I think this is best left to individual language clients to make recommendations.

Not a blocker as I believe language clients can always make their recommendations, and this OTEP already has an explicit call out about by passing OTel Log API and leveraging language existing API.

Reply:
The primary reason I can think of is because it would allow the adopter (application), vender (distro) to configure their usage of events in whatever manner they would like without needing to create duplicate code to intercept any Log API or Log Processor or Logging framework to be able to redirect the events they care about to do whatever they want with them...

Reply:
Irrespective of whether OTel Log API or 3rd party API is used, routing can be done in exact same way right? Both produce a LogRecord with EventName in it, with no way to distinguish between who/how produced it.

Reply:
> If the 'other api' has support for emitting structured logs with event name, should Otel have reason to recommend one over the other? I think this is best left to individual language clients to make recommendations.

are you thinking of the case (e.g. Rust and .Net), where the "other API" will be explicitly adopted as _the_ OpenTelemetry Log API? if so, that will be the one we recommend (since it will be _the_ OpenTelemetry Log API)

Reply:
Only OTel .NET has make such adoption. OTel Rust **recommends** a particular existing logging API, nothing more. (so far). 

In case of OTel Rust/others, if the existing API can very well produce OTel Events (LogRecord with name and schema), any reason why we should recommend OTel owns API over other?

Logging libraries get improved all the time, so even the ones that are not good today may improve over time and might very well gain the ability to produce OTel Events. eg: Golang's native logging became `slog` and added structured logging recently. .NET's ILogger gained ability to automatically add EventName recently. 

Hence the qn- why OTel as a whole has to recommend our own API, vs leaving that to individual language clients?

If Rust's logging library evolves like that, and we can totally use that in instrumentations we maintain. I actually prefer to recommend people use Rust's existing logging library then instead of OTel's. The native logging libraries have other advantages that OTel probably don't want to chase and match.